starknet_devnet/util.py

```diff
@@ -123,3 +157,16 @@ def __init__(self):
         self.retdata = []
         self.internal_calls = []
         self.l2_to_l1_messages = []
+
+def enable_pickling():
```

Comment:
Why is this a method and not done inside of the `StraknetContract`?

Reply:
This is a monkey patch. `StarknetContract` is not our class, it's from starkware's library. By default it cannot be pickled, so this patch was necessary.